src/helpers/interfaces.ts

```diff
@@ -9,8 +9,12 @@ import {
   CI_STAGE_NAME,
   CI_WORKSPACE_PATH,
   GIT_BRANCH,
+  GIT_COMMIT_AUTHOR_DATE,
   GIT_COMMIT_AUTHOR_EMAIL,
   GIT_COMMIT_AUTHOR_NAME,
+  GIT_COMMIT_COMMITTER_DATE,
+  GIT_COMMIT_COMMITTER_EMAIL,
+  GIT_COMMIT_COMMITTER_NAME,
```

Comment:
shouldn't we update getGitMetadata() to read these new tags from the .git folder?

Reply:
it does already, but you made me realise that we gain from adding an output type to `getGitMetadata`